### DISCUSSION TITLE
how to enable ipsec for specific port

### DISCUSSION THREAD

**Discussion by dewilllll** [strongswan/strongswan#1576] [Q&A]
Hi Guys,

I want to enable ipsec for a specific port, I have 2 connections on my Linux machine one is for management and other I want to use for Ipsec connection with windows host.
I need to use start action as trap for which disconnect from both networks,
I want to enable for only on one port.

[br_edif_swanctl.txt](https://github.com/strongswan/strongswan/files/10896166/br_edif_swanctl.txt)

**Comment by tobiasbrunner** [comment]
Sorry, I don't understand the question. Could you please update it and clarify what exactly it is you want to do and what you mean with "port"?

**Reply by dewilllll** [reply]
Actually, I mean ethernet port, I have 2 ips one is for management/ public ip and other is private ip on which i want to establish ipsec connection

**Reply by tobiasbrunner** [reply]
And what's the problem exactly?

**Reply by dewilllll** [reply]
when I load the config, I lost the connection/ access of both IPs. That i dont want to happen

**Reply by tobiasbrunner** [reply]
Consider configuring `local|remote_ts` then (or configure a single address in `remote_addrs`). Configuring a subnet in `remote_addrs` with `start_action=trap` and `mode=transport` triggers a [special mode](https://www.strongswan.org/testing/testresults/ikev2/trap-any/), but without any traffic selectors this will result in policies with `0.0.0.0/0` on both ends, i.e. they capture all traffic.
